.github/workflows/build-package.yml.template

```
@@ -109,16 +112,30 @@ jobs:
       - name: Tag release
         run: |
           (
+            set -euxo pipefail
             cd trigger
-            git tag $CRATE-$VERSION-$TARGET_ARCH
-            git push origin $CRATE-$VERSION-$TARGET_ARCH
+            git tag -f $CRATE-$VERSION $CRATE-$VERSION-$TARGET_ARCH
```

we definitely don't want to replace tags that already exist
```suggestion
            git tag $CRATE-$VERSION || echo "$CRATE-$VERSIONtag already exists"
            git tag $CRATE-$VERSION-$TARGET_ARCH
```

---

@alsuren What if the `$CRATE-$VERSION-$TARGET_ARCH` already exists but `$CRATE-$VERSION` does not?

---

```suggestion
            git tag $CRATE-$VERSION || echo "$CRATE-$VERSION tag already exists"
            git tag $CRATE-$VERSION-$TARGET_ARCH || echo "$CRATE-$VERSION-$TARGET_ARCH tag already exists"
```